src/AppBundle/Controller/ChatPageController.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace AppBundle\Controller;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Response;
+
+class ChatPageController extends Controller
+{
+    public function pageAction()
+    {
+        return new Response(
```

Comment:
it is no need to wrap `$this->rendor` into `new Response()` you just could use `return $this->render('chat.html.twig');`